android/src/main/java/com/rnziparchive/RNZipArchivePackage.java

```diff
@@ -18,6 +18,11 @@
   }
 
   @Override
+  public List<Class<? extends JavaScriptModule>> createJSModules() {
+    return Collections.emptyList();
+  }
+
```

Comment:
@plrthink Oh, how odd. I couldn't compile my test app unless I added that back in again. Must be a versioning issue in my setup somewhere. Sorry about that.